impeller/renderer/backend/metal/allocator_mtl.mm

```diff
@@ -88,6 +90,18 @@ static bool SupportsLossyTextureCompression(id<MTLDevice> device) {
 #endif
 }
 
+void DebugAllocatorStats::Increment(size_t size) {
+  size_.fetch_add(size, std::memory_order_relaxed);
+}
+
+void DebugAllocatorStats::Decrement(size_t size) {
+  size_.fetch_sub(size, std::memory_order_relaxed);
+}
+
+size_t DebugAllocatorStats::GetAllocationSizeMB() {
+  return size_ * 1e-6;
```

Comment:
nit: this is doing a conversion between 64bit floats and 64bit integers.

Reply:
What change are you requesting?

Reply:
friendly ping @gaaclarke

Reply:
It's not a huge deal but divide by `1_000_000` instead.

Reply:
Done!